storage_service/storage_service/settings/production.py

```diff
@@ -4,53 +4,45 @@
 from os import environ
 
 from .base import *  # noqa: F401, F403
-from .base import get_env_variable  # To make he linter happy
-
-# Normally you should not import ANYTHING from Django directly
-# into your settings, but ImproperlyConfigured is an exception.
-from django.core.exceptions import ImproperlyConfigured
-
-
-
-
-def get_env_setting(setting):
-    """ Get the environment setting or return exception """
-    try:
-        return environ[setting]
-    except KeyError:
-        error_msg = "Set the %s env variable" % setting
-        raise ImproperlyConfigured(error_msg)
-
 
 # ######## HOST CONFIGURATION
 # See: https://docs.djangoproject.com/en/1.5/releases/1.5/#allowed-hosts-required-in-production
 ALLOWED_HOSTS = ['*']
 # ######## END HOST CONFIGURATION
 
 # ######## EMAIL CONFIGURATION
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-backend
-EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
 
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-host
-EMAIL_HOST = environ.get('EMAIL_HOST', 'smtp.gmail.com')
+#
+# Which Email backend to use? SMTP is default, others including Amazon SES are
+# also supported.  See https://docs.djangoproject.com/en/dev/ref/settings/ and
+# https://docs.djangoproject.com/en/dev/topics/email/
+#
 
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-host-password
-EMAIL_HOST_PASSWORD = environ.get('EMAIL_HOST_PASSWORD', '')
+EMAIL_BACKEND = environ.get('EMAIL_BACKEND',
+    'django.core.mail.backends.smtp.EmailBackend')
+
+# Amazon SES Backend
+# See https://github.com/azavea/django-amazon-ses
+DJANGO_AMAZON_SES_REGION = environ.get('DJANGO_AMAZON_SES_REGION', 'us-east-1')
 
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-host-user
-EMAIL_HOST_USER = environ.get('EMAIL_HOST_USER', 'your_email@example.com')
 
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-port
-EMAIL_PORT = environ.get('EMAIL_PORT', 587)
+# SMTP Backend
+EMAIL_HOST = environ.get('EMAIL_HOST', 'smtp.gmail.com')
+EMAIL_HOST_PASSWORD = environ.get('EMAIL_HOST_PASSWORD', '')
+EMAIL_HOST_USER = environ.get('EMAIL_HOST_USER', 'your_email@example.com')
+EMAIL_PORT = to_int(environ.get('EMAIL_PORT', '587'))
+EMAIL_SSL_CERTFILE = environ.get('EMAIL_SSL_CERTFILE')
+EMAIL_SSL_KEYFILE = environ.get('EMAIL_SSL_KEYFILE')
+EMAIL_USE_SSL = is_true(environ.get('EMAIL_USE_SSL'))
```

Comment:
Since `is_true` expects a string, can this be given a default value?

Reply:
Ah yes, I just realised that `is_true` will error if its `env_str` parameter is `None`. The default for the `EMAIL_USE_SSL` setting is `False` so I'll change it to use that. Is it worth also defending `is_true` against `None` input?